packages/jest-haste-map/src/crawlers/__tests__/watchman.test.js

```diff
@@ -117,73 +122,57 @@ describe('watchman watch', () => {
       expect(calls[0][0][0]).toEqual('watch-project');
       expect(calls[1][0][0]).toEqual('watch-project');
 
-      // Calls 2 and 3 are queries
-      const query1 = calls[2][0];
-      const query2 = calls[3][0];
-      expect(query1[0]).toEqual('query');
-      expect(query2[0]).toEqual('query');
+      // Call 2 is the query
+      const query = calls[2][0];
+      expect(query[0]).toEqual('query');
 
-      expect(query1[2].expression).toEqual([
-        'allof',
-        ['type', 'f'],
-        ['anyof', ['suffix', 'js'], ['suffix', 'json']],
-        ['anyof', ['dirname', ROOT_MOCK + FRUITS]],
-      ]);
-      expect(query2[2].expression).toEqual([
+      expect(query[2].expression).toEqual([
         'allof',
         ['type', 'f'],
         ['anyof', ['suffix', 'js'], ['suffix', 'json']],
-        ['anyof', ['dirname', ROOT_MOCK + VEGETABLES]],
+        ['anyof', ['dirname', 'fruits'], ['dirname', 'vegetables']],
       ]);
 
-      expect(query1[2].fields).toEqual(['name', 'exists', 'mtime_ms']);
-      expect(query2[2].fields).toEqual(['name', 'exists', 'mtime_ms']);
+      expect(query[2].fields).toEqual(['name', 'exists', 'mtime_ms']);
 
-      expect(query1[2].suffix).toEqual(['js', 'json']);
-      expect(query2[2].suffix).toEqual(['js', 'json']);
+      expect(query[2].suffix).toEqual(['js', 'json']);
 
       expect(data.clocks).toEqual({
-        [FRUITS]: 'c:fake-clock:1',
-        [VEGETABLES]: 'c:fake-clock:2',
+        [ROOT_MOCK]: 'c:fake-clock:1',
       });
 
       expect(data.files).toEqual(mockFiles);
 
-      path.relative = originalPathRelative;
-
       expect(client.end).toBeCalled();
     });
   });
 
-  it('updates the file object when the clock is given', () => {
+  test('updates the file object when the clock is given', () => {
```

Comment:
why did you switch to `test` instead of `it`?

Reply:
`it` is not directly documented and seems like the preference is to use `test` instead.